examples/013_octaves_comparison.py

```diff
@@ -0,0 +1,398 @@
+#
+# Octave bands are frequency bands that have a width of one octave. This means that an octave
+# band is always twice as wide as the previous octave band and half as wide as the successive
+# octave band.
+#
+# One-third-ctave bands are frequency bands built from the subdivision of octave bands into three
+# sections.
+#
+# ANSI S1.11-1986 and IEC 61260:1995 standards define the band limits and the corresponding
+# filter sets to apply in order to compute the level within these 29 bands from the temporal
+# signal.
+#
+# Why are there several computation methods, and when to use them?
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# There are several methods to compute octave and one-third-octave levels in PyAnsys Sound, which
+# differ by the type of input data used and by the computation method.
+# More details about the computation can be found in the :ref:`Theory` section below.
+#
+# :class:`OctaveLevelsFromSignal`
+#     This class can be used only if a time-domain signal is available as input.
+#     It will produce a result following the ANSI S1.11-1986 and IEC 61260:1995 standards, and can
+#     therefore be used when a standard result is needed.
+#
+# :class:`OneThirdOctaveLevelsFromSignal`
+#     This class can be used with a Power Spectral Density (PSD).
+#     This PSD may be computed from a temporal signal or provided directly as the result of a
+#     simulation.
+# * It should be used with parameter ``use_ansi_s1_11_1986=False`` (default), when there is no need
+#   to compare with results computed on a time-domain signal.
+#
+# * It should be used with parameter ``use_ansi_s1_11_1986=True``, when the user wants to compare
+#   with results computed on a time-domain signal, or with standard results.
+
+
+# %%
+# Octave computation
+# ~~~~~~~~~~~~~~~~~~
+# Here we show how to compute octave levels using :class:`OctaveLevelsFromSignal`.
+#
+# The same processing can be applied using :class:`OctaveLevelsFromPSD`,
+# :class:`OneThirdOctaveLevelsFromSignal` and :class:`OneThirdOctaveLevelsFromPSD`.
+
+# %%
+# Set up analysis: load Ansys libraries, connect to the DPF server, and
+# download the necessary data files.
+
+# Load standard libraries.
+import matplotlib.pyplot as plt
+
+# Load Ansys libraries.
+from ansys.sound.core.examples_helpers import (
+    download_flute_wav,
+)
+from ansys.sound.core.server_helpers import connect_to_or_start_server
+from ansys.sound.core.signal_utilities import LoadWav
+from ansys.sound.core.spectral_processing import PowerSpectralDensity
+from ansys.sound.core.standard_levels import (
+    OctaveLevelsFromSignal,
+    OneThirdOctaveLevelsFromPSD,
+    OneThirdOctaveLevelsFromSignal,
+)
+
+# Connect to a DPF Sound server (remote or local).
+my_server = connect_to_or_start_server(
+    ansys_path=r"C:\ANSYSDev\Tests\DPF Servers\20251215\ansys\dpf\server_2026_1_pre1",
+    use_license_context=True,
+)
```

Comment:
@anshlachamb a remaining hard-coded path here

```suggestion
my_server = connect_to_or_start_server(
    use_license_context=True,
)
```